src/main/java/frc/robot/subsystems/IndexerSubsystem.java

```diff
@@ -1,48 +1,94 @@
 package frc.robot.subsystems;
 
+import static edu.wpi.first.units.Units.Amps;
 import static frc.robot.Constants.IndexerConstants.DEVICE_ID_BELT;
 import static frc.robot.Constants.IndexerConstants.EJECT_VELOCITY;
 import static frc.robot.Constants.IndexerConstants.INTAKE_VELOCITY;
 import static frc.robot.Constants.IndexerConstants.SCORE_VELOCITY_LEVEL_1;
+import static frc.robot.Constants.IndexerConstants.SLOT_CONFIGS;
+import static frc.robot.Constants.IndexerConstants.SUPPLY_CURRENT_LIMIT;
 
+import com.ctre.phoenix6.SignalLogger;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.TorqueCurrentFOC;
 import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
 import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Subsystem;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine.Direction;
 
-/*
+/**
  * The subsystem for the coral box/indexer
  */
 public class IndexerSubsystem implements Subsystem {
 
   private final TalonFX beltMotor = new TalonFX(DEVICE_ID_BELT);
   private final VelocityTorqueCurrentFOC beltControl = new VelocityTorqueCurrentFOC(0.0);
+  private final TorqueCurrentFOC beltSysIdControl = new TorqueCurrentFOC(0.0);
+
+  private final SysIdRoutine beltSysIdRoutine = new SysIdRoutine(
+      new SysIdRoutine.Config(null, null, null, state -> SignalLogger.writeString("Indexer SysId", state.toString())),
+      null);
```

Comment:
This doesn't look right, a `Mechanism` should get passed here, not `null`. I discovered this from the warning that `beltSysIdControl` is unused, we should be keeping an eye on the warnings (ideally there are none.)